app/src/androidTest/java/com/github/se/project/ui/profile/CreateTutorProfileTest.kt

```diff
@@ -38,21 +40,24 @@ class CreateTutorProfileTest {
         Mockito.`when`(currentProfile).thenReturn(MutableStateFlow<Profile?>(null))
       }
 
+  // Helper function to create a mock Profile
+  private fun getMockProfile() =
+      Profile(
+          uid = "1",
+          googleUid = "googleUid",
+          firstName = "First",
+          lastName = "Last",
+          phoneNumber = "1234567890",
+          role = Role.TUTOR,
+          section = Section.GM,
+          academicLevel = AcademicLevel.MA2,
+          languages = listOf(),
+          subjects = listOf(),
+          schedule = listOf())
+
```

Comment:
Great idea to define the `MockProfile` outside of the test, as it helps eliminate repetition and keeps the test code cleaner.